src/System.Management.Automation/resources/RunspaceStrings.resx

```diff
@@ -202,7 +202,7 @@
     <value>There is no disconnected command associated with this runspace.</value>
   </data>
   <data name="DisconnectNotSupportedOnServer" xml:space="preserve">
-    <value>The disconnection operation is not supported on the remote computer. To support disconnecting, the remote computer must be running Windows PowerShell 3.0 or a later version of Windows PowerShell.</value>
+    <value>The disconnection operation is not supported on the remote computer. To support disconnecting, the remote computer must be running Windows PowerShell 3.0 or a later version of Windows PowerShell and using the WSMan transport.</value>
```

Comment:
From the message I understand nothing that I should do.
Maybe simply say "The disconnection operation is not supported for SSH sessions."

Reply:
It's also not supported for named pipes and PowerShell Direct remoting.  The doc update I linked in the PR description should hopefully make this more clear.